docs/production/pcbs.md

```diff
@@ -14,7 +20,7 @@ However, the Gerber export is a bit more complicated than providing native files
 
 After the board is opened in Pcbnew, the solder mask clearance and min width must be set to zero. This is important because the manufacturers normally want to define the clearance themselves based on their process. Setting it to zero makes it easier for them to shrink/enlarge the pads for the solder mask. The dialogue can be found under *File > Board Setup* as shown in Figure 1.
 
-<fig-caption src="production/pcbs_kicad_pads_clearance.png" caption="Pads and Mask Clearance dialog (KiCad v5.1.4)" num="1" />
+<fig-caption src="production/pcbs_kicad_pads_clearance.png" caption="Pads and Mask Clearance dialog (KiCad v5.1.4)" num="2" />
```

Comment:
clearance

Reply:
What do you mean by clearance? (I see that the figure number is wrong in the text, though. will fix that)

Reply:
In the caption its written with capital "C", is that intended?

Reply:
Ah, right. No, not really. Will fix it and also write mask with lower-case letter.